src/Extensions/Abstractions/OmexServiceContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Information about service context</summary>
+	public abstract class OmexServiceContext : IServiceContext
+	{
+		/// <summary>Creates an instance of OmexServiceContext</summary>
```

Comment:
>summary [](start = 7, length = 7)

Here and throughout, `<param>` and `<return>` comments should also be added where appropriate

Reply:
I think here and in most of the cases those tags are redundant and don't add value since I'll just duplicate parameter name or return type of method